spec/models/admin_user_spec.rb

```diff
@@ -0,0 +1,14 @@
+require 'rails_helper'
+
+RSpec.describe AdminUser, type: :model do
+  describe 'db table' do
+    it {is_expected.to have_db_column :email}
+    it {is_expected.to have_db_column :encrypted_password}
+  end
+
+  describe "Factory" do
+    it "should have a valid factory" do
+      expect(create(:user)).to be_valid
```

Comment:
The name of the factory is ```admin_user``` and not ```user``` :) You are checking the wrong factory.

Reply:
good catch, idk why i had that in the first place